src/diagnostics/eng/Versions.props

```diff
@@ -114,8 +114,8 @@
   </PropertyGroup>
   <ItemGroup Condition="!$(PrivateBuildTesting) AND !$(InternalReleaseTesting)">
     <RuntimeTestVersions Include="Latest">
-      <RuntimeDownload>$(VSRedistCommonNetCoreSharedFrameworkx64100Version)</RuntimeDownload>
-      <Runtime>$(MicrosoftNETCoreAppRuntimewinx64Version)</Runtime>
+      <RuntimeDownload>$(MicrosoftNETCorePlatformsVersion)</RuntimeDownload>
```

Comment:
Why the platforms package? Just arbitrary?

Reply:
The platforms package is a non-shipping package (so non-stable version even in stable version scenarios) that's built on all verticals.

Reply:
It doesn't stabilize, and is a package that I don't expect to go away, become stable, etc. in the future. We could add a formal sentinel package to runtime, but I think it's unnecessary.